src/components/CardView/CardView.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+
+const CardView = (props) => {
+  const { title, text, imageUrl, imageAlt, buttonUrl, buttonText } = props;
+
+  let imageEle;
+  if (imageUrl) {
+    imageEle = <img className="card-img-top" src={imageUrl} alt={imageAlt} />;
+  }
+
+  let buttonEle;
+  if (buttonUrl && buttonText) {
+    buttonEle = <a href={buttonUrl} className="btn btn-secondary" target="_blank">{buttonText}</a>;
+  }
+
+  return (
+    <div className="card">
+      {imageEle}
+      <div className="card-block">
+        <h4 className="card-title">{title}</h4>
+        <p className="card-text">{text}</p>
```

Comment:
can we follow BEM convention for classes?

e.g. `card__block`, `card__title` etc